app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -210,6 +210,17 @@ class NavigationDrawerTestActivityTest {
     }
   }
 
+  @Test
+  fun testNavigationDrawerTestActivity_openNavigationDrawer_selectSwitchProfileMenu_showsExitToProfileChooserDialog_clickCancel_checkDrawerIsClosed(){
+    launch(NavigationDrawerTestActivity::class.java).use {
+      onView(withId(R.id.home_activity_drawer_layout)).perform(open())
+      onView(withText(R.string.menu_switch_profile)).perform(click())
+      onView(withText(R.string.home_activity_back_dialog_message)).check(matches(isDisplayed()))
+      onView(withText(R.string.home_activity_back_dialog_cancel)).perform(click())
+      onView(withId(R.id.home_activity_drawer_layout)).check(matches(isClosed()))
```

Comment:
You will need to add one more line to check that `Home` is highlighted when you open drawer again. That was one of the main issues that we are fixing here. Also, you will need to update your test case name accordingly

Reply:
Changes done @rt4914

Reply:
You are checking the content on home fragment, that is incorrect. We need to check that the navigation drawer has "Home" as highlighted.

Reply:
@rt4914 I have made the change

Reply:
Actually this is better, but I had something else in mind. But i believe we cannot write the test case that I am thinking until we have Screenshot testing.